lib/src/test/kotlin/starknet/data/TypedDataTest.kt

```diff
@@ -524,7 +524,7 @@ internal class TypedDataTest {
     }
 
     @Nested
-    inner class InvalidTypesTest {
+    inner class TypesTest {
```

Comment:
A there was need to test resolved revision correctness (from domain object bot as string and int), I've decided to rename `InvalidTypesTest` to `TypesTest` and broaden its scope.
There is also a possibility to keep `InvalidTypesTest` and create separate eg. `ValidTypesTest` class. This way we will keep current test score and create add a new one. Wdyt? @DelevoXDG @THenry14

Reply:
> A there was need to test resolved revision correctness (from domain object bot as string and int), I've decided to rename InvalidTypesTest to TypesTest and broaden its scope.

`InvalidTypesTests` is a scope to deal with **types** specifically, as in types (basic/preset/custom) used in typed data. So tests related to `revision` don't really fit there.

We should instead:
1. Remove `domain v1 object with integer revision`
2. Move `domainObjectV1WithStringRevision` inside  `domain v1 object with string revision` test
3. Move `domain v1 object with string revision` outside of `InvalidTypesTest`

> There is also a possibility to keep InvalidTypesTest and create separate eg. ValidTypesTest class.

I think this should be just its own separate test. You can look for another scope to put it, but definitely not anything with `Types` in it.

Reply:
Sure, that's why I've added remark about it. I will move it to separate test then.